### PR TITLE
Update table, Using aggregates, 

### PR DESCRIPTION
**Pull request by EddyShimwa**
### In this step i used the queries to solve all the questions in the requirements as it is shown in the following screenshots 

image

image

digi poki mon

savepoint

unspecified

count

AVG

ghh

hyyy

last

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -15,5 +15,28 @@ SELECT name FROM animals WHERE name <> 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+-- Day two
+
+UPDATE animals
+SET species = CASE
```
Comment (Davidosky007) [inline queries.sql:26]: * As required in the [project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species%20column%20to%20unspecified.%20Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.) you are to 

> update the animals table by setting the species column to unspecified

> Verify that the change was made on the animal table 

> Then roll back the change and verify that the species columns went back to the state before the transaction. Kindly work on all these 🙏

File: queries.sql [hunk 2]
```diff
@@ -15,5 +15,28 @@ SELECT name FROM animals WHERE name <> 'Gabumon';
...
+WHEN name LIKE '%mon' THEN 'digimon'
+ELSE 'pokemon'END
```
Comment (Davidosky007) [inline queries.sql:28]: * As stated in the requirement 

> verify that the change was made after updating animals' table 

> verify that change persists after commit, Kindly work on these 🙏

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @EddyShimwa 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights

* Well written PR description 👍 
* Proper use of Gitflow 👍 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (EddyShimwa) [reply queries.sql:28]: Thanks @Davidosky007 for reviewing this PR!

Comment (EddyShimwa) [reply queries.sql:28]: Here! I did all the changes suggested by the previous reviewer!

**Comment by EddyShimwa** [comment]
Here I made all the changes suggested tby the previous reviewer!

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand:  @EddyShimwa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :green_circle: 

- PR is professional :+1: 



## Optional suggestions
**N/A**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by EddyShimwa** [comment]
Thanks @FrederickMih for reviewing this PR!

**Pull request merged**